File: haushaltsbuch/src/test/java/org/github/haschi/haushaltsbuch/Kontozeile.java
package org.github.haschi.haushaltsbuch;

import cucumber.deps.com.thoughtworks.xstream.annotations.XStreamConverter;
import org.github.haschi.haushaltsbuch.api.Buchung;
import org.github.haschi.infrastruktur.BuchungConverter;

class Kontozeile
{
    @XStreamConverter(BuchungConverter.class)
    Buchung soll;

    @XStreamConverter(BuchungConverter.class)
    Buchung haben;
}

File: haushaltsbuch/src/test/kotlin/org/github/haschi/haushaltsbuch/Kontozeile.kt
package org.github.haschi.haushaltsbuch

import cucumber.deps.com.thoughtworks.xstream.annotations.XStreamConverter
import org.github.haschi.haushaltsbuch.core.Buchung
import org.github.haschi.infrastruktur.BuchungConverter

class Kontozeile {
    @XStreamConverter(BuchungConverter::class)
    var soll: Buchung? = null

    @XStreamConverter(BuchungConverter::class)
    var haben: Buchung? = null
}
